app/services/event_service.rb

```diff
@@ -0,0 +1,16 @@
+class EventService
+
+  def self.find_all(zipcode)
```

Comment:
this method is getting pretty long. i'd suggest maybe breaking this up a bit. for example, `conn` could be it's own method because there's a high likelihood (maybe not in this assessment, but in general) that we'd be making more than one request to this endpoint with perhaps different parameters 🎋